www/src/pages/table/NodeSearchComponent.tsx

```diff
@@ -71,9 +73,7 @@ export const NodeSearchComponent = withStyles(styles)(
         type: "class",
         id: "",
         label: (
-          <div>
```

Comment:
remove the div tags and just use the Spin component available from ant-d. hence, the custom SpinComponent can be removed.

Reply:
We are using the Css we add in that component